pcdsdevices/sxr_test_absorber.py

```diff
@@ -34,10 +34,13 @@ def set(
         moved_cb: Callable | None = None,
         timeout: float | None = None,
     ):
+
+        """
         if self.st3k4_auto.get():
-            raise ST3K4AutoError(
-                "ST1K4 must follow ST3K4. Move rejected."
-            )
+           raise ST3K4AutoError(
+               "ST1K4 must follow ST3K4. Move rejected."
+           )
+        """
         return super().set(position, moved_cb=moved_cb, timeout=timeout)
```

Comment:
I think the correct fix here is to completely remove this `set` method override, starting from `def set` and ending at this line.

If we no longer need to prevent the moves here, we don't need this method at all, we can use the parent class's set instead.